### DISCUSSION TITLE
Superset fails to install

### DISCUSSION THREAD

**Discussion by esuess** [dat-linux/community#27] [Issues]
I am interested installing Superset.  There is an error message about pyenv: version `3.9.13' not installed. 

Any suggested solution to this problem?  

Thank you

**Comment by dat-linux** [comment]
Hi, pyenv tool should be pre-installed at `$HOME/.pyenv/bin/pyenv`

If not run:

`git clone https://github.com/pyenv/pyenv.git  $HOME/.pyenv`

When running `$HOME/.pyenv/bin/pyenv local` you should see "3.9.13"

If this fails please run:

`$HOME/.pyenv/bin/pyenv install 3.9.13`

**Reply by dat-linux** [reply]
However, this should all have be done during the Superset installation process. At what point are you seeing this error message? Please post any output to help trouble shooting, thanks.

**Comment by esuess** [comment]
The new install script from Control Panel works to install Superset.  Thank you.

**Reply by dat-linux** [reply]
Thanks for the feedback.
